neptune/new/internal/init_impl.py

```diff
@@ -84,6 +85,94 @@ def init(project: Optional[str] = None,
          monitoring_namespace: str = "monitoring",
          flush_period: float = 5,
          proxies: dict = None) -> Run:
+    """Starts a new tracked run, and append it to the top of the Runs table view.
+
+    Args:
+        project(str, optional): Name of a project in a form of namespace/project_name. Defaults to `None`.
```

Comment:
namespace/project_name -> 'namespace/project_name'
str -> `str`

Reply:
This comment also applies to other occurrences of types and strings.

Reply:
str -> `str`

If str is in braces it should be one of:
* (str)
* (:obj:`str`)

So it's good.